### PR TITLE
Feature: Add option to show card IDs always

### LINKED ISSUE #21: Add option to allways see card ID
Now need to hove on card to see it ID. Please add an option to always show all card IDs on board. Also allow to see ID in opened card.

### PR DESCRIPTION
**Pull request by Rj7**
Thank you for a great add on @christiankaindl . 

Added two options. One to show card IDs always and another to show only on hover. By default set to previous behaviour of showing only on hover.  Please review and let me know if any changes. 

Partially fixes #21

### CODE REVIEW

File: settings/options.html [hunk 1]
```diff
@@ -51,7 +51,11 @@ <h1 data-translate="settingsHeading">Preferences</h1>
       <section>
         <div class="checkbox">
           <input id="trello-id" type="checkbox" name="copyId" checked>
-          <label for="trello-id" data-translate="settingsIdTitle">Card IDs on hover</label>
+          <label for="trello-id" data-translate="settingsIdTitle">Show card ID</label>
+          <div>
+            <input class="secondary" id="trello-id-hover" type="checkbox" name="copyIdHover" />
+            <label class="secondary" for="trello-id-hover" data-translate="settingsIdTitleHover" >Show only on hover</label>
```
Comment (christiankaindl) [inline settings/options.html]:
```suggestion
            <label class="secondary" for="trello-id-hover" data-translate="settingsIdTitleHover" >Only show IDs on hover</label>
```

File: _locales/en/messages.json [hunk 2]
```diff
@@ -36,7 +36,11 @@
     "description": "Description of the visible label text feature."
   },
   "settingsIdTitle": {
-    "message": "Card IDs on hover",
+    "message": "Show card ID",
```
Comment (christiankaindl) [inline _locales/en/messages.json]: Mising **s**

```suggestion
    "message": "Show card IDs",
```

File: _locales/en/messages.json [hunk 3]
```diff
@@ -36,7 +36,11 @@
...
+    "description": "Show the Card IDs always."
```
Comment (christiankaindl) [inline _locales/en/messages.json]:
```suggestion
    "description": "Name of the Show Card IDs feature."
```

**Comment by Rj7** [comment]
@christiankaindl  Thanks for the review. I have made the changes and remove other language translations.  Please have a look at it.

**Comment by christiankaindl** [comment]
Thanks! I'll have a look at it later today and will test it thoroughly. Then I can merge it and release a new version, hopefully today or tomorrow.

**Comment by christiankaindl** [comment]
**The new version 0.12.0 is now live! https://addons.mozilla.org/en-US/firefox/addon/trello-super-powers/**

You can update via Firefox's Add-on page (type about:addons in the url bar, or click on Firefox's menu in the top right and choose "Add-ons").

Thank you so much @Rj7 for your contribution :star: 

Quick question: Was the code well structured for you to dive in? Or is there something that should be documented for other contributors?

**Comment by Rj7** [comment]
@christiankaindl Thanks :+1: . That's my first open source contribution live that I use daily. 
And for the code, Yes it was structured well and I was able to able to dive into the code pretty quickly. I will let know if I find anything particularly noticeable.

**Pull request merged**